sync/sync.go

```diff
@@ -229,6 +240,7 @@ func (s *Synchronizer) verifierTask(ctx context.Context, block *core.Block, stat
 			}
 
 			s.newHeads.Send(block.Header)
+			s.newReceipts.Send(block.Receipts[0])
```

Comment:
Need to move this somewhere where it will make more sense and send the actual transaction receipt for the given transaction hash. Any ideas?